### ISSUE TITLE
docs.nestjs.cn is down, and the CDN of jsdelivr cannot be accessed in China, resulting in inaccessibility

### ISSUE THREAD

**Issue by podywu** [nestjs/nest#9725]
### Current behavior

docs.nestjs.cn is down, and the CDN of jsdelivr cannot be accessed in China, resulting in inaccessibility.

### Expected behavior

expect docs.nestjs.cn is normal to access, thanks.

**Comment by Tony133** [comment]
The site: `docs.nestjs.cn` it seems to work

<img width="1463" alt="screenshot" src="https://user-images.githubusercontent.com/11542387/172007770-3eb2b304-9a90-42aa-bd53-f5d88674a01a.png">

**Comment by kamilmysliwiec** [comment]
docs.nestjs.cn is not maintained by the official organization
